### ISSUE TITLE
Go Pulumi SDK - Helm error

### ISSUE THREAD

**Issue by HeberGB** [pulumi/pulumi-kubernetes#1427]
### Problem description

I been trying to create a new chart and also I tried to add the [example chart](https://www.pulumi.com/docs/reference/pkg/kubernetes/helm/v3/chart/#remote-chart) but always I got the same error: `program failed: cannot marshal an input of type *v1.ClusterRole as a value of type pulumi.Resource`

### Errors & Logs

![image](https://user-images.githubusercontent.com/18607092/104779604-fbdeb000-5744-11eb-8aa3-a7ca53de6c7f.png)

### Reproducing the issue

My setup is
```go
	_, err = helmv3.NewChart(ctx, "aws-ebs-csi-driver", helmv3.ChartArgs{
		Namespace: pulumi.String("kube-system"),
		Chart:     pulumi.String("aws-ebs-csi-driver"),
		Values: pulumi.Map{
			"enableVolumeScheduling": pulumi.Bool(true),
			"enableVolumeResizing":   pulumi.Bool(true),
			"enableVolumeSnapshot":   pulumi.Bool(true),
		},
		FetchArgs: helmv3.FetchArgs{
			Repo: pulumi.String("https://kubernetes-sigs.github.io/aws-ebs-csi-driver"),
		},
	})
	if err != nil {
		return err
	}
```

**Comment by lukehoban** [comment]
This sounds like the same issue as https://github.com/pulumi/pulumi-kubernetes/issues/1397, but that was fixed in `v2.7.5`.  Can you confirm you are using a version higher than that?

**Comment by lblackstone** [comment]
Never heard back, so assuming this was fixed.
